forem/forem
# Use preact-modal in adminModal

## What type of PR is this? (check all applicable)


## Description

This PR replaces regular modal with preact-modal in adminModal file.

## Related Tickets & Documents



- Related Issue #16328
- Closes #16328

## QA Instructions, Screenshots, Recordings

**How to test this?**
1. Sign in as admin and visit http://localhost:3000/admin/customization/config
2. Click on `Authentication`
3. Click `Enable`, do not enter keys.
4. Click `Update Settings`
5. A modal will appear with warning


All three-instances where modal is getting used.

<img width="600" alt="Screenshot 2022-10-25 at 2 17 58 PM" src="https://user-images.githubusercontent.com/9396084/197728471-618aba8a-d608-464c-bf8d-3236c7c74650.png">

<img width="600" alt="Screenshot 2022-10-26 at 2 26 25 PM" src="https://user-images.githubusercontent.com/9396084/197982127-8147a9a2-c4d6-4f44-b717-2c085076c447.png">

<img width="600" alt="Screenshot 2022-10-26 at 2 26 41 PM" src="https://user-images.githubusercontent.com/9396084/197982143-0cce6a76-b901-42b8-8e2a-9e657a449024.png">


### UI accessibility concerns?

None

## Added/updated tests?


## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_
